pkg/kubelet/dockertools/manager_test.go

```diff
@@ -1822,21 +1824,20 @@ func TestCheckVersionCompatibility(t *testing.T) {
 		// Invalid apiversion
 		{"invalid_api_version", false},
 	}
-	for i := range apiVersion {
-		apiVersion[i]++
+	for i := range intSlice {
```

Comment:
I think listing some explicit test cases is enough. This test makes me want to write a test to validate it ...